README.md

```diff
@@ -26,7 +26,7 @@ Source code contoh program bisa diunduh di [github.com/novalagung/dasarpemrogram
 
 ## Kontribusi
 
-Ebook ini merupakan project open source, teruntuk siapapun yang ingin berkontribusi silakan langsung saja cek [github.com/novalagung/dasarpemrogramanrust](https://github.com/novalagung/dasarpemrogramanrust). Cek juga [halaman kontributor](/CONTRIBUTING) untuk melihat list kontributor.
+Ebook ini merupakan project open source, teruntuk siapapun yang ingin berkontribusi silakan langsung saja cek [github.com/novalagung/dasarpemrogramanrust](https://github.com/novalagung/dasarpemrogramanrust). Cek juga [halaman kontributor](/docs/CONTRIBUTING.md) untuk melihat list kontributor.
```

Comment:
actually the correct path is [/CONTRIBUTING](https://dasarpemrogramanrust.novalagung.com/CONTRIBUTING). the `/docs` prefix is rewritten into `/` and no need to specify the `.md` extension

the `/docs/CONTRIBUTING.md` generates 404 error 
![image](https://user-images.githubusercontent.com/982868/209933805-3ef02a2d-086f-43a9-8a31-5f4719a6d92f.png)

Reply:
I'm sorry, I thought the README for docs and for repository is different, because when I click on contributor page in repository it will lead to 404 page

I return to the previous code (https://github.com/novalagung/dasarpemrogramanrust/pull/17/commits/2c4f6e1e5ca6e1dcfff1f0f896c8138d495fa487), if I find a solution a special pull request will be made